geometa/rest.py

```diff
@@ -58,10 +58,22 @@ def get_documents_by_radius(user, latitude, longitude, radius):
     return _find(user, query)
 
 
+def _get_path_after_download(girder_file):
+    with NamedTemporaryFile(delete=False) as f:
```

Comment:
In this case, the temporary file should be deleted after the job is complete. Ideally this context manager would wrap the full lifecycle of the file, then we won't need `delete=False`.

Reply:
@zachmullen I addressed your comment. Let me know what you think.